.github/workflows/projects.yml

```diff
@@ -0,0 +1,133 @@
+name: GitHub Projects Automations
+permissions: {}
+
+on:
+  issues:
+    types:
+      - assigned
+      - labeled
+
+  pull_request_target:
+    types:
+      - assigned
+      - labeled
+      - opened
+
+env:
+  ISSUE_URL: ${{ github.event.issue.html_url || github.event.pull_request.html_url }}
+
+jobs:
+  team_board:
+    name: Provider Team Project
+    needs: community_check
+    runs-on: ubuntu-latest
+
+    env:
+      PROJECT_ID: PVT_kwDOAAuecM4AF-7h
+      PROJECT_NUMBER: 196
+      STATUS_FIELD_ID: PVTSSF_lADOAAuecM4AF-7hzgDcsQA
+      STATUS_IN_PROGRESS: ${{ vars.team_project_status_in_progress }}
+      VIEW_FIELD_ID: PVTSSF_lADOAAuecM4AF-7hzgMRB34
+      VIEW_ENGINEERING_INITIATIVE: ${{ vars.team_project_view_engineering_initiative }}
+      VIEW_MAINTAINER_PR: ${{ vars.team_project_status_maintainer_pr }}
+      VIEW_WORKING_BOARD: ${{ vars.team_project_view_working_board }}
+
+    steps:
+      - name: Generate GitHub Authentication Token
+        id: token
+        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a # v2.1.0
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          installation_retrieval_mode: id
+          installation_retrieval_payload: ${{ secrets.INSTALLATION_ID }}
+          private_key: ${{ secrets.APP_PEM }}
+          permissions: >-
+            {
+              "organization_projects": "write"
+            }
+
+      - name: Check if Author is Maintainer
+        id: author
+        if: github.event.action == 'opened'
+        env:
+          MAINTAINERS: ${{ secrets.MAINTAINERS }}
+          AUTHOR_LOGIN: ${{ github.event.pull_request.user.login }}
+        run: |
+          echo "is_maintainer=$(echo $MAINTAINERS | base64 --decode | jq --arg u $AUTHOR_LOGIN '. | contains([$u])')" >> "$GITHUB_OUTPUT"
```

Comment:
Doing "community check" with this `bash` one-liner is the same amount of lines as using the reusable workflow (if you don't count the `env` block that could _technically_ be eliminated, to the detriment of readability, in my opinion).